cdap-watchdog/src/main/java/co/cask/cdap/metrics/query/MetricsDiscoveryHandler.java

```diff
@@ -140,6 +146,17 @@ public MetricsDiscoveryHandler(Authenticator authenticator, final MetricsTableFa
         return map;
       }
     });
+
+    this.timeSeriesTables = Suppliers.memoize(new Supplier<Map<MetricsScope, TimeSeriesTable>>() {
+      @Override
+      public Map<MetricsScope, TimeSeriesTable> get() {
+        Map<MetricsScope, TimeSeriesTable> map = Maps.newHashMap();
+        for (final MetricsScope scope : MetricsScope.values()) {
+          map.put(scope, metricsTableFactory.createTimeSeries(scope.name(), SECONDS_RESOLUTION));
```

Comment:
so we always use highest resolution? Will it be more efficient if we use lowest? Or we may miss some data when searching that way?